sdk/ai/azure-ai-projects/azure/ai/projects/tracing/__init__.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
```

Comment:
I think a better place for this would be under the /agents/tracing folder, since all files in this folder are all about Agents.

Reply:
moved there